.github/workflows/Fox_water_recovery.yml

```
@@ -0,0 +1,218 @@
+jobs:
+  build:
+    name: Build OFRP by ${{ github.actor }}
+    runs-on: [self-hosted, linux, x64, gpu]
+    if: github.event.repository.owner.id == github.event.sender.id
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    permissions:
+      contents: write
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)